src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -501,6 +512,15 @@ private void updateButtons() {
             downloadButton.setVisible(true);
             cancelDownloadButton.setVisible(false);
         }
+
+        // if less than 200MB available
+        if (downloadDirectory.getUsableSpace() <= 200 * 1000000) {
```

Comment:
I tried to do a bit of research and this seems to be quite a contentious issue... There seem to be 2 units (1 megabyte = 1000 bytes, 1 mebibyte = 1024 bytes) and different platforms seem to use different units (Windows uses mebibytes, Mac uses megabytes). No idea which one would better.... 😐